specific/cmo_stock/models/stock_valuation_history.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+from openerp import models, fields, tools
+
+
+class StockHistory(models.Model):
+    _inherit = 'stock.history'
+
+    partner_id = fields.Many2one(
+        'res.partner',
+        string='Partner',
+    )
+    operating_unit_id = fields.Many2one(
+        'operating.unit',
+        string='Operating Unit',
+    )
+
+    # Replace
+    def init(self, cr):
+        tools.drop_view_if_exists(cr, 'stock_history')
```

Comment:
I don't know about adding fields in the function create view of sql, please recommend.
Introduction, I add comment in  the header function.